src/__tests__/Search.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import Search from "../components/Search";
+
+describe("Search", () => {
+    const { asFragment } = render(<Search />);
```

Comment:
There is an error here when trying to run the tests caused by not passing in a required prop to your component on line 6. Could that be added in?

Also, could another test be added here to check the handler function is called when the button is clicked?